test/factories/cms.rb

```diff
@@ -51,15 +51,17 @@
   end
 
   factory(:cms_page, :parent => :cms_template, :class => CMS::Page) do
-    # association is copied to child factory because we reference provider
-    # and it is not yet created, but copying association to this factory fixes it
-    association :provider, :factory => :provider_account
```

Comment:
this seems to be outdated because factories work alright without it

Reply:
`:cms_template` already defines this association. Probably in the past there was some inheritance issue and it had to be duplicated here as well.